guides/common/modules/proc_setting-a-default-encrypted-root-password.adoc

```diff
@@ -12,7 +12,7 @@ If you change the password and reprovision the hosts in the group that inherits
 . Generate an encrypted password:
 +
 -----------------
-# python -c 'import crypt,getpass;pw=getpass.getpass(); print(crypt.crypt(pw)) if (pw==getpass.getpass("Confirm: ")) else exit()'
+# python3 -c 'import crypt,getpass;pw=getpass.getpass(); print(crypt.crypt(pw)) if (pw==getpass.getpass("Confirm: ")) else exit()'
 -----------------
```

Comment:
```suggestion
-----------------
+
[NOTE]
====
On some older systems, such as {EL} 7, you must use `python` instead of `python3`.
====
```

Reply:
I'm not sure if that's true. Are you sure?

Reply:
Yes, I had to deal with this in the downstream API guide, see https://gitlab.cee.redhat.com/satellite-6-documentation/docs-Red_Hat_Satellite_6/-/merge_requests/1747

Reply:
Cool, thanks!

Reply:
I am not sure if this hint is really necessary here; your call.

Reply:
Actually, on [RHEL 7.7+, Python 3 is available](https://access.redhat.com/solutions/2353081) alongside Python 2. I'm not adding the note.